crates/trigger/src/runtime_config/sqlite.rs

```diff
@@ -49,24 +49,35 @@ async fn execute_statements(
     if statements.is_empty() {
         return Ok(());
     }
-    let Some(default) = databases.get("default") else {
-        debug_assert!(
-            false,
-            "the 'default' sqlite database should always be available but for some reason was not"
-        );
-        return Ok(());
-    };
 
     for m in statements {
-        if let Some(file) = m.strip_prefix('@') {
+        if let Some(config) = m.strip_prefix('@') {
```

Comment:
For me the argument parser is now complicated enough to merit pulling out into a function (ideally with a few simple tests).  I know some folks have different tolerances for this though, especially as you'd probably need to add an enum for the "file vs literal SQL" distinction.  Definitely not a blocker!